src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonConverterOfT.cs

```diff
@@ -17,6 +17,19 @@ public abstract partial class JsonConverter<T> : JsonConverter
         /// When overidden, constructs a new <see cref="JsonConverter{T}"/> instance.
         /// </summary>
         protected internal JsonConverter()
+        {
+            Initialize();
+        }
+
+        internal JsonConverter(bool initialize)
```

Comment:
What use case requires passing `initialize: false`? If any modifications are required to any of the computed properties, in most cases we just override them in the derived constructors.